marketplace/core/types/externals/omie/views.py

```diff
@@ -1,6 +1,9 @@
 from rest_framework.decorators import action
 from rest_framework.response import Response
 
+from marketplace.connect.client import ConnectProjectClient
```

Comment:
`import ConnectProjectClient` Not being used, for better visualization unused imports are removed